app/components/Footer/index.css

```diff
@@ -56,72 +60,92 @@
       }
     }
     & .footer-contact {
```

Comment:
The whole contact box (input, envelope, button) is pretty janky on mobile (alignments and paddings are missing)